documentation/helpers/ToC/useActiveAnchor.js

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react'
```

Comment:
Why not use TS? 😢

Reply:
Because it was supposed to be a task that took 5 minutes by using SB's Table of Content feature. But unfortunately I couldn't, I'll update to TS :D